src/Engine.php

```diff
@@ -163,15 +163,9 @@ public function search(string $query, string $indexUid, array $searchParams): ar
 
     /**
      * Search the index and returns the number of results.
-     *
-     * @param string $query
-     * @param string $indexUid
-     * @param array  $requestOptions
-     *
-     * @return int
```

Comment:
Is there a reason to remove the comment about params and return?

Reply:
The doc block does not provide any helpful information as the method is fully type-hinted.
As some other PR removes all the doc blocks that are there, although the method is fully type-hinted, I already removed it here.